.github/workflows/deploy-example.yml

```diff
@@ -0,0 +1,77 @@
+name: Deploy Signer and Dapp apps.
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ["main", "production"]
+
+jobs:
+  deploy:
+    name: Deploy
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: npm
+
+      - name: Install DFX
+        uses: aviate-labs/setup-dfx@v0.2.6
+        with:
+          dfx-version: 0.15.3
+          dfx-disable-encryption: true
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Format Check
+        run: npm run format:check
+
+      - name: Build
+        run: npm run build
+
+      - name: Test
+        run: npm run test
```

Comment:
From my point of view, we could remove them bc they had been running on PR already. And `dfx deploy` will run `npm run build` automatically, as far as I know.